.github/workflows/rust_container_publishing.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     push_to_registry:
       name: Push Docker image to Docker Hub
       runs-on: ubuntu-latest
-      if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
Is this because creating a release circumvents the workflow run?

Reply:
Not circumvents, but doesn't have one. The workflow run happens on push, disconnected from the release.